code/index.html

```diff
@@ -1,30 +1,114 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+  <title>calm down and subscribe</title>
+</head>
+
+<body>
+  <!--header with hero video-->
+  <header class="video-header">
+    <video autoplay loop muted>
+      <source src="./videos/Sunset.mp4" type="video/mp4">
+    </video>
+    <h1>sign up and learn how to relax the mind</h1>
+  </header>
+
+  <!--container with registration and faq-->
+  <div class="container">
+
+    <!--form container-->
+    <div class="card-container">
+
+      <div class="content">
+        <form action="https://httpbin.org/anything" method="POST">
+
+          <!--table-->
+          <table>
+            <h2>Registration</h2>
+            <div class="input-box">
+              <label for="customer-name">Full name </label>
+              <input type="text" name="customer_name" placeholder="Full Name" id="name" required>
+            </div>
+
+            <div class="input-box">
+              <label for="email">Email </label>
+              <input type="email" name="customer_email" placeholder="Email" id="email" required>
+            </div>
+
+            <div class="input-box">
+              <label for="customer_number">Phone Number </label>
+              <input type="number" name="customer_number" placeholder="Phone Number" id="number" required>
+            </div>
+
+            <div class="input-box">
+              <label for="email_address">Password </label>
+              <input type="password" name="email_address" placeholder="Password" id="password" required>
```

Comment:
Here you could write password instead of email_address (the for and name attributes).